src/System.Management/tests/WmiTestHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Management.Tests
+{
+    public static class WmiTestHelper
+    {
+        private static readonly bool s_ElevatedAndNotNanoServer = AdminHelpers.IsProcessElevated() && PlatformDetection.IsNotWindowsNanoServer;
+        private static readonly string s_systemDriveId = Environment.GetEnvironmentVariable("SystemDrive");
+
+        public static string Namespace => "root/WmiEBvt";
+        public static string SystemDriveId => s_systemDriveId;
```

Comment:
why do you need the privates ones also?

Reply:
`IsProcessElevated` is the main reason since it needs to open and check the process token, this way it is called only once, since I wanted to do it for it, I did it for all of them.